src/components/App.js

```diff
@@ -54,42 +54,41 @@ class App extends Component {
     }).catch((err) => alert(`You hit a problem ${err}`))
   }
 
-  componentWillMount() {
-    let data = bookContract;
-    // this.getBookData("0316067598"); These can be used as inputs to test
-    // this.getBookData("9780060830939");
-    
+  addContract = (contract) => {
+    localStorage.setItem('contract', contract);
     this.setState({
-      isAvailable: String(data.isAvailable()),
-      owner: String(data.owner())
-    });
+      bookchainContract: localStorage.contract
+    })
   }
-
+  // componentWillMount() {
+  //   this.setState({bookchainContract: localStorage.contract})
+  // }
 
   render() {
+    let form = null
+    let wallet = null
+    if (this.state.bookchainContract) {
+      form = <BookForm getBookData={this.getBookData}/>
+      wallet = <div> Wallet ID = {this.state.bookchainContract} </div>
+    } else {
+      form = <ContractForm addContract={this.addContract} />
+    }
+
+
     return (
       <div className="App">
         <div className="App-header">
           <img src={logo} className="svg-logo" alt="logo" />
           <h2>Bookchain</h2>
         </div>
-        <div className="App-intro">
+        <div className="body">
```

Comment:
I would suggest changing the name to something that would never conflict with the html defaults.